forms-html/evan-hanan/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <form action="#">
+        <label for="email">Enter your email</label> <br>
+        <input type="email" name="email" id="email" required placeholder="Enter email"> <br>
+        <label for="username">Username:</label><br>
+        <input type="text" id="username" name="username"><br>
+        <label for="password">Password</label><br>
+        <input type="password" id="password" name="password"><br>
+        <label for="quantity"> Quantity (between 1 and 5)</label><br>
+        <input type="number" id="quantity" name="quantity" min="1" max="5"><br>
+        <label for="range">Volume(between 0 and 50)</label><br>
+        <input type="range" id="range" name="range"><br>
+        <br>
+        <br>
+        <p>radio</p>
+        <input type="radio" id="radio1" name="radio">
+        <label for="radio1">Male</label> <br>
+        <input type="radio" id="radio2" name="radio">
+        <label for="radio2">Female</label> <br>
+        <input type="radio" id="radio3" name="radio">
+
+        <label for="radio3">Others</label><br>
```

Comment:
Make sure that your Radio inputs have a value attribute. For more info, check out:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio